src/chaos-experiments/models/database/sequelize/sequelizeConnector.js

```diff
@@ -159,7 +159,7 @@ async function getChaosJobExperimentById(jobExperimentId, contextId) {
     return chaosExperiment;
 }
 
-async function getChaosJobExperimentByJobId(jobId, contextId) {
+async function getChaosJobExperimentsByJobId(jobId, contextId) {
```

Comment:
Why did this function changed? It's not the same meaning getting by id and by job id